README.md

```diff
@@ -2,7 +2,7 @@
 This project lets you run a small localhost server that communicates with stellar network.
 It will create a KIN asset on Stellar Test Network, related to the issuer seed in `config.json` file.
 You will have a rest-api `sendkin` where you need to pass address as url parameter<br/>
```

Comment:
> a rest-api `sendkin`

you mean a function that submits a REST HTTP request?

Reply:
also what is that `<br />` at the end

Reply:
* yup
* new line